Source/Notifications/ChangeCalculation/ChangedIndexes.swift

```diff
@@ -29,18 +28,18 @@ extension NSOrderedSet {
     }
 }
 
-extension Array where Element : Hashable {
+extension Array where Element: Hashable {
 
     public func toOrderedSetState() -> OrderedSetState<Element> {
         return OrderedSetState(array: self)
     }
 }
 
-public struct OrderedSetState<T: Hashable> : Equatable {
+public struct OrderedSetState<T: Hashable>: Equatable {
+
+    public private(set) var array: [T]
+    public private(set) var order: [T : Int]
```

Comment:
```suggestion
    public private(set) var order: [T: Int]
```
Shouldn't this be like this?